src/FFI.php

```diff
@@ -252,37 +287,25 @@ private static function init(): void
 
         $is_64bits = PHP_INT_SIZE === 8;
 
-        $libraryPaths = [
-            "" // system library
-        ];
-
-        $vipshome = getenv("VIPSHOME");
-        if ($vipshome) {
-            // lib<qual>/ predicates lib/
-            $libraryPaths[] = $vipshome . ($is_64bits ? "/lib64/" : "/lib32/");
-            // lib/ is always searched
-            $libraryPaths[] = $vipshome . "/lib/";
-        }
-
         if (PHP_OS_FAMILY === "OSX" || PHP_OS_FAMILY === "Darwin") {
             // Homebrew on Apple Silicon
-            $libraryPaths[] = "/opt/homebrew/lib/";
+            self::$libraryPaths[] = "/opt/homebrew/lib";
```

Comment:
Maybe `self::addLibraryPath("/opt/homebrew/lib");`? Same for other direct write to `$libraryPath[]`.